### PR TITLE
fixed --tap-device value

### PR DESCRIPTION
**Pull request by mehrdadrad**
Signed-off-by: Mehrdad Arshad Rad <arshad.rad@gmail.com>

Issue #, if available:

Description of changes:
the value of the tap-device is tap interface/mac address but it was veth

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/network-setup.md [hunk 1]
```diff
@@ -58,8 +58,7 @@ curl -X PUT \
     }'
 ```
 
-Alternatively, if you are using firectl, add `--tap-
-device=veth0/AA:FC:00:00:00:01` to your command line.
+Alternatively, if you are using firectl, add `--tap-device=tap0/AA:FC:00:00:00:01` to your command line.
```
Comment (aghecenco) [inline docs/network-setup.md]: Please add a newline after `--tap-` as it was earlier.

Comment (mehrdadrad) [reply docs/network-setup.md]: @aghecenco sure but why it needs newline? the key is --tap-device=

Comment (andreeaflorescu) [reply docs/network-setup.md]: We are keeping all lines less than 80 characters. That's why you need the newline. You could also have it like:
```
Alternatively, if you are using firectl, add 
`--tap-device=tap0/AA:FC:00:00:00:01` to your command line. 
```

Comment (mehrdadrad) [reply docs/network-setup.md]: @andreeaflorescu thanks, I just fixed it.

**Comment by mehrdadrad** [comment]
@aghecenco I squashed them anything else I should do ....

**Comment by andreeaflorescu** [comment]
@mehrdadrad you still have 3 commits on this branch. One is generated from the update on the branch and the other 2 are yours. Can you squash the two commits, please?

**Pull request closed without merging**